py/desispec/qa/qa_quicklook.py

```diff
@@ -120,7 +121,7 @@ def run_qa(self,image,paname=None,amps=False,qafile=None, qafig=None,qlf=False):
 
         if qafile is not None:
             f=open(qafile,"w")
-            yaml.dump(retval,f)
+            f.write(yaml.dump(yamlify(retval)))
```

Comment:
I should think it would be better to have QA I/O handled
by desispec.io.qa

Consider it.